src/components/code/ComponentPreview.tsx

```diff
@@ -33,21 +34,27 @@ const formatName = (path: string) => {
  *
  * @param {string} path - The path to the component relative to `src/showcase`.
  * @param {boolean} usingFramer - Whether the component is using Framer Motion.
+ * @param {React.ReactNode} preview - Optional preview to render instead of the component.
  * Example: "components/button/3DButton"
  *
  * Usage: <ComponentPreview path="components/button/3DButton" />
+ *
```

Comment:
in both lines can you add: 

`usage with path` and `usage with preview`

and this can be changed to: 
` * @param {React.ReactNode} preview - optional preview to render a component directly instead of using path.
`

Reply:
Good idea. Just updated the documentation